grpc-interface-support/README.md

```diff
@@ -123,3 +125,24 @@ int main(int argc, char** argv) {
 }
 
 ```
+
+***Python***
+
+Generated files:
+
+* `<Service-Name>ServiceStub.py` contains class definition and method declarations for implementing the server. This file is continuously auto-generated! To add custom methods place them inside the `<Service-Name>ServiceImpl.py` class.
+* `<Service-Name>ServiceImpl.py` contains method definitions for all server methods, defaulting to `UNIMPLEMENTED`. This file is only generated **once** if it does not exist.
+
+```python
+from seats_service_sdk.seats_pb2_grpc import SeatsServicer
+from seats_service_sdk.SeatsServiceServerFactory import SeatsServiceServerFactory
+
+async def on_start(self):
+    server = SeatsServiceServerFactory.create(
+        SeatsServicer(),
+        MyMiddleware(), # Create your own middleware sub class
```

Comment:
There should be sub-classes already. `NativeMiddleware` is used by default, unless the SDK is configured differently: https://github.com/eclipse-velocitas/vehicle-app-python-sdk/blob/main/velocitas_sdk/config.py
try:
```python

from velocitas_sdk.config import middleware

<...>
    server = SeatsServiceServerFactory.create(
        SeatsServicer(),
        middleware
```